test_single_controller_ppo.py

```diff
@@ -358,6 +346,10 @@ def setup_process_groups(
 class TrainActorGroup(SPMDActorGroup):
     """Group of training actors for PPO."""
 
+    def __init__(self, *args: Any, **kwargs: Any):
+        super().__init__(*args, **kwargs)
+        self.experience_buffer = None
```

Comment:
unused? I don't think the ActorGroup. needs to hold onto the experience buffer

Reply:
Oops forgot to delete this, thanks for catching it